docs/source/addons/i18n.md

```diff
@@ -29,7 +29,7 @@ it and add it into your project's workflow.
 An specific `src/i18n.js` script has been added to the addon generator in order to do this task.
 You can find it here:
 
-https://github.com/plone/volto/blob/master/packages/generator-volto/generators/addon/templates/src/i18n.js
+https://github.com/plone/volto/blob/master/packages/generator-volto/generators/addon/templates/src/index.js
```

Comment:
This is not i18n script afaik. The script now lives under `@plone/scripts` package: https://github.com/plone/volto/blob/master/packages/scripts/i18n.js
-cc @sneridagh

Reply:
@ksuess cleaned this up.